components/script/dom/mod.rs

```diff
@@ -324,11 +324,7 @@ pub(crate) mod fontfaceset;
 pub(crate) mod formdata;
 pub(crate) mod formdataevent;
 pub(crate) mod gamepad;
-pub(crate) mod gamepadbutton;
-pub(crate) mod gamepadbuttonlist;
-pub(crate) mod gamepadevent;
-pub(crate) mod gamepadhapticactuator;
-pub(crate) mod gamepadpose;
+pub(crate) use self::gamepad::*;
```

Comment:
Should we remove it as well?

Reply:
The generated bindings assume that all dom interfaces exist at crate::dom::$name.